membership-attribute-service/app/services/IdentityAuthService.scala

```diff
@@ -22,8 +22,9 @@ class IdentityAuthService(apiConfig: IdapiConfig)(implicit ec: ExecutionContext)
       .map(user => Option(user))
       .handleError { err =>
         if(err.isInstanceOf[UserCredentialsMissingError])
-          SafeLogger.warn(s"invalid request as no token or cookie provided - unable to authorize user. \n" +
-            s"Request attempted: ${requestHeader.method} ${requestHeader.path} with headers:\n ${requestHeader.headers.toMap.mkString("\n")}", err)
```

Comment:
This was introduced in https://github.com/guardian/members-data-api/pull/397 as a temporary measure for debugging.

Reply:
Logging the request headers in this case, I mean